Sources/TokamakCore/Views/TextField.swift

```diff
@@ -36,8 +34,8 @@ extension TextField where Label == Text {
   ) where S: StringProtocol {
     label = Text(title)
     textBinding = text.projectedValue
-    editingChangedAction = onEditingChanged
-    commitAction = onCommit
+    self.onEditingChanged = onEditingChanged
+    self.onCommit = onCommit
   }
 
   // FIXME: implement this method, which uses a Formatter to control the value of the TextField
```

Comment:
<!--
  1 failure:  FIXMEs should be ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/TokamakCore/Views/TextField.swift;
  Line: 41;
-->
- :no_entry_sign: FIXMEs should be resolved (implement this method, which u...). (`todo`)